awscli/customizations/s3/subcommands.py

```diff
@@ -540,8 +541,11 @@ def _handle_rm_force(self, parsed_globals, parameters):
         # to be kept simple.
         if 'force' in parameters:
             if parameters['force']:
-                bucket = find_bucket_key(parameters['src'][5:])[0]
-                path = 's3://' + bucket
+                bucket = find_bucket_key(parameters['src'][5:])
+                if bucket[1]:
```

Comment:
This line is fine. The find_bucket_key returns a tuple of two elements, the bucket and the key. So ``bucket[1]`` results in returning the value for the key. So for example take a look at this snippet:
```py
In [1]: from awscli.customizations.s3.utils import find_bucket_key

In [2]: find_bucket_key('mybucket/')
Out[2]: ('mybucket', '')

In [3]: find_bucket_key('mybucket')
Out[3]: ('mybucket', '')

In [4]: find_bucket_key('mybucket/mykey')
Out[4]: ('mybucket', 'mykey')
```
If you further tried to index ``bucket[1]``, you may run into an IndexError, but you do not. So that is fine.

One thing that you could do to make it more clear is make the ``find_bucket_key`` line the following:
```py
bucket, key = find_bucket_key(parameters['src'][5:])
if key:
    ...
```
Keeping it the name bucket is a little misleading because in reality it is a tuple. Otherwise, I am fine with this PR. I can merge it unless you want to change the line. Let me know what you think.